src/wechaty.ts

```diff
@@ -409,8 +536,14 @@ export class Wechaty extends EventEmitter implements Sayable {
   }
 
   /**
-   * get current user
-   * @returns {Contact} current logined user
+   * Get current user
+   *
+   * @returns {Contact}
+   * @example
+   * ```ts
+   * const contact = Wechaty.self()
```

Comment:
Should not use `Wechaty` because it's the class name.

should instantiate an instance for Wechaty, like wechaty, and use `bot` in the example code.

```ts
const bot = Wechaty.instance()
```